js/Ice/greeter/.npmrc

```diff
@@ -0,0 +1,2 @@
+# Use ZeroC nightly registry for @zeroc packages
+@zeroc:registry=https://download.zeroc.com/nexus/repository/npm-nightly/
```

Comment:
This instructs NPM to lookup packages with `@zeroc` scope in the given registry.